src/renderers/PartialViewRenderers/region/componentsInfo.js

```diff
@@ -170,65 +183,36 @@ function getFacilitiesList() {
     $('input[type=radio][name=usedComponentDisplay]').on('change', e => {
         e.preventDefault();
-        rbsComponentsDisplayer();
+        filterComponents()
     });
 
     $('input[type=radio][name=activatedComponentDisplay]').on('change', e => {
         e.preventDefault();
-        rbsComponentsDisplayer();
+        filterComponents()
     });
 }
 
-/**
- * Display component's rows based on radio button filter:
- * Used/Unused, Activated/Unactivated Components
- */
-function rbsComponentsDisplayer() {
-    let usedComponent = $('input[name=usedComponentDisplay]:checked').val();
-
-    let componentGetterEvent = 'Component:getComponentList';
-    let componentReceiverEvent = 'Component:getComponentListOK';
-    if (usedComponent === 'used') {
-        componentGetterEvent = 'Component:getUsedComponentList';
-        componentReceiverEvent = 'Component:getUsedComponentListOK';
-    } else if (usedComponent === 'unused') {
-        componentGetterEvent = 'Component:getUnusedComponentList';
-        componentReceiverEvent = 'Component:getUnusedComponentListOK';
+function filterComponents() {
+    let usedComponentInput = $('input[name=usedComponentDisplay]:checked').val();
+    let activatedComponentInput = $('input[name=activatedComponentDisplay]:checked').val();  
+    let componentList = JSON.parse(localStorage.getItem(COMPONENT_LIST))
+
+    if(usedComponentInput === USED){
+        componentList = componentList.filter(component=> component.facilityId !== null)
+    }else
+    if(usedComponentInput === UNUSED){
+        componentList = componentList.filter(component=> component.facilityId === null)
     }
 
-    ipcRenderer.send(componentGetterEvent, parseInt(getProcessArgObj()));
-    ipcRenderer.once(componentReceiverEvent, (e, res) => {
-        setComponentList(res);
-        filterComponents();
-    });
-}
-
-/**
- * Filter component's table list based on search string filter 
- * and whether the component is activated or not
- */
-function filterComponents() {
-    let searchString = $('#txtComponentSearch').val().toLowerCase();
-    let activatedComponent = $('input[name=activatedComponentDisplay]:checked').val();
-    let showActivated = (activatedComponent === 'activated');
-    let hiddenRowIds = [];
-
-    $('#componentsList tr').not('#componentTemplateRow').each(function (i, row) {
-        const componentData = $(this).data('componentData');
-        let doShowActivated = (activatedComponent === 'all') ||
-                              (showActivated && componentData.activationTime <= 0) ||
-                              (!showActivated && componentData.activationTime > 0);
-        let doShow = ($(this).text().toLowerCase().indexOf(searchString) > -1) && doShowActivated;
-        if (doShow) {
-            $(this).show();
-        } else {
-            hiddenRowIds.push(`#${row.id}`);
-            $(this).hide();
-        }
-    });
+    if(activatedComponentInput === ACTIVATED){
```

Comment:
Same here, a more descriptive name perhaps? Feel free to chat me to brainstorm name ideas :v